scripts/utils/sign_and_send.js

```diff
@@ -73,7 +67,44 @@ module.exports = function (web3 = web3, artifacts = artifacts) {
     console.log("Transaction awaiting execution in the interface", interfaceLink)
   }
 
+  /**
+   * Checks that a transaction was already proposed to the gnosis-safe UI
+   *
+   * @param {Address} masterSafe Address of the master safe owning the brackets
+   * @param {Transaction} transaction The transaction to be signed and sent
+   * @param {string} network either rinkeby or mainnet
+   * @param {number} [nonce=null] specified transaction index. Will fetch correct value if not specified.
```

Comment:
Nitpick, the index is a different number in the web interface.
```suggestion
   * @param {number} [nonce=null] specified Gnosis Safe transaction nonce. Will fetch correct value if not specified.
```